install_config/monitoring/update-and-compatibility-guarantees.adoc

```diff
@@ -0,0 +1,20 @@
+[[update-and-compatibility-guarantees]]
+= Update and compatibility guarantees
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+In order to be able to deliver updates with guaranteed compatibility, configurability of the OpenShift Monitoring stack is limited to the explicitly available options. This document describes known pitfalls of which types of configuration and customization are unsupported, as well as misuse of resources provided by OpenShift Monitoring. All configuration options described in xref:configuring-openshift-cluster-monitoring.adoc#configuring-openshift-cluster-monitoring[Configuring OpenShift cluster monitoring] are explicitly supported. See also xref:configuring-openshift-cluster-monitoring.adoc#supported-configuration[Supported configuration].
```

Comment:
The Configuring OpenShift cluster monitoring and Supported configuration links are broken.